### ISSUE TITLE
Possible to input motion data directly to move playspace?

### ISSUE THREAD

**Issue by eilerj2** [naelstrof/VRPlayspaceMover#16]
Wondering if it is possible to provide motion data input to VRPlayspaceMover without using a Vive tracker. Looking to compensate for motion from a simulator that has the ability to output its motion data.

**Comment by naelstrof** [comment]
Yeah! The source code is fully available, there's no officially supported way as of right now though.
Though you probably should just use Open Input Emulator's functionality to set the position of headset and controllers.

**Comment by eilerj2** [comment]
Thank you very much for the reply. Would it be possible at all for you to help me walk through this process or send links where walkthroughs exist? This is my first adventure into modding my VR setup.
